WordPress/Classes/ViewRelated/Blog/Blog Details/Detail Header/SiteIconView.swift

```diff
@@ -33,7 +33,8 @@ class SiteIconView: UIView {
     }()
 
     let activityIndicator: UIActivityIndicatorView = {
-        let indicatorView = UIActivityIndicatorView(style: .large)
+        let indicatorView = UIActivityIndicatorView(style: .medium)
+        indicatorView.color = .white
```

Comment:
Just a small change that fixes the spinner style. We reduced the size of the site icon earlier but never updated the spinner.